WeakAuras/Prototypes.lua

```diff
@@ -4493,6 +4493,7 @@ WeakAuras.event_prototypes = {
                 destName = UnitExists(destUnit) and UnitName(destUnit) or "",
                 show = true,
                 changed = true,
+                inverse = true,
```

Comment:
I'm pretty sure inverse depended on whether the cast was channeling or not.

Reply:
you are right